judge/views/user.py

```diff
@@ -335,6 +338,59 @@ def get(self, request, *args, **kwargs):
         return response
 
 
+class WCIPEGMergeActivate(LoginRequiredMixin, TitleMixin, FormView):
+    template_name = 'user/wcipeg-merge-activate.html'
+    form_class = WCIPEGMergeActivationForm
+
+    def merge(self, from_user, to_user):
+        from_user.merge_user = to_user
```

Comment:
I think we should move this inside the transaction, since we don't want the `merge_user` to be set if the objects fail to be migrated over.